homeassistant/components/sabnzbd/__init__.py

```diff
@@ -24,12 +24,15 @@
 
 _CONFIGURING = {}
 
+CONF_URLBASE = 'urlbase'
```

Comment:
`url_base` or I think `base_path` makes more sense? `url_base` is too close to base url

Reply:
This was chosen to be consistent with Sonarr and Radarr modules (as you're highly likely to be using some combination of them.